src/main/java/org/opensearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -522,9 +529,13 @@ public Set<String> getAllConfiguredTenantNames() {
 
     public boolean multitenancyEnabled() {
         return privilegesInterceptor.getClass() != PrivilegesInterceptor.class
-                && dcm.isDashboardsMultitenancyEnabled();
+                && tcm.isDashboardsMultitenancyEnabled();
```

Comment:
Can we remove this method from DynamicConfigModel and its inheritor DynamicConfigModelV7 since it is not used anymore?